packages/app-store/zapier/api/subscriptions/listBookings.ts

```diff
@@ -24,6 +24,9 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       take: 3,
       where: {
         userId: validKey.userId,
+        eventType: {
```

Comment:
The userId of the key does not need to match if it is a team key. Like that, it would only list team bookings of the user who also created the key not all team bookings

We do that in many other places as well

Reply:
I think in the DB and and API key relationship with a User is required. Should we allow API keys to be linked to only a team?